Src/Support/Google.Apis.Auth/OAuth2/GoogleAuthConsts.cs

```diff
@@ -37,7 +37,7 @@ public static class GoogleAuthConsts
         public const string ApprovalUrl = "https://accounts.google.com/o/oauth2/approval";
 
         /// <summary>The authorization token server URL.</summary>
-        public const string TokenUrl = "https://accounts.google.com/o/oauth2/token";
+        public const string TokenUrl = "https://oauth2.googleapis.com/token";
```

Comment:
I'm not completely sure we should do this. See the comment on the `OidcTokenUrl` property.

Reply:
Aha. Hadn't seen that. So we'll always use the new one by default already... that's probably good enough.

Reply:
Yes, the old one (this one) isn't used by any of our code.

Reply:
Righto. Will revert this file when I get a chance. It's probably worth us fixing the test code though.

Reply:
Agreed :)